data/backup-script/backup.sh

```diff
@@ -0,0 +1,10 @@
+echo 'Backup Influx metadata'
```

Comment:
This can't live in data by default. It needs to live in the directory with the docker file, and it needs to be copied into the second instance's data tree. Please review the docker documentation for how to install files while building the docker machine.  The data directory is only accidentally in the same directory (i.e., by default) -- it can be anywhere, potentially on a different volume. The design of docker requires that this kind of thing be constructed and populated via docker, otherwise permission, etc., will be wrong.

Reply:
I now see that, in fact, this is a mistaken commit, as it's a duplicate of the script below.